en/docs/tutorials/scenarios/scenario-overview.md

```diff
@@ -2,6 +2,8 @@
 
 **Union Station** is a major multimodal railway transportation hub. It is one of the busiest stations in the country and serves thousands of passengers a day. The train shed, platforms, and tracks are owned by **GOGO transit**, and they operate the station. Trains are owned by the companies named **Quantis**, **ColTrain**, and **RailCo**. To provide a digital ecosystem, all four companies are planning to develop their day to day business operations with WSO2 technology. These development ranges from providing different kinds of APIs to external/internal users, providing real time notifications, stream data processing, integrating with partners/external systems etc.
 
+![Scenario Overview]({{base_path}}/assets/img/tutorials/scenario-tutorials/scenario-overview.png)
```

Comment:
Should we change this to the clickable image format?

Reply:
Will fix in the next PR. Using the same branch so might push too many other changes in with that fix.